shotover-proxy/src/protocols/cassandra_protocol2.rs

```diff
@@ -649,8 +679,17 @@ fn get_cassandra_frame(rf: RawFrame) -> Result<Frame> {
     }
 }
 
+impl CodecErrorFixup for CassandraCodec2 {
+    fn fixup_err(
+        &self,
+        message: Message,
+    ) -> (Option<Message>, Option<Message>, Option<anyhow::Error>) {
+        (Some(message), None, None)
```

Comment:
Is the idea that you will put the error handling in here but you just havent done so yet?